src/main/webapp/i18n/de/exam.json

```diff
@@ -141,12 +142,26 @@
             "notConfirmed": "Bitte setze einen Haken in das Kästchen, um fortzufahren.",
             "falseName": "Der angegebene Name ist nicht korrekt. Bitte versuche es erneut!",
             "notSet": "Nicht gesetzt",
             "startExamToolTip": "Button wird 5 Minuten vor der Klausur aktiviert.",
+            "welcomePageText": {
+                "readCarefully": "Bitte lese die folgenden Informationen sorgfältig!",
+                "workOnYourOwn": "Du musst die Klausur eigenständig bearbeiten. Diese Klausur ist eine Open-Book-Klausur (einschließlich der Nutzung des Internets), aber du <span class='fw-bold text-primary'>darfst in keiner Weise</span> Unterstützung von anderen Personen erhalten (persönlich, per Chat, Forum, Diskussionsgruppe, künstliche Intelligenz, etc.). Andernfalls gilt dies als <span class='fw-bold text-primary'>Täuschung</span> (\"Unterschleif\") und führt zu den in der APSO (\"Allgemeine Prüfungs- und Studienordnung\") genannten Konsequenzen. Insbesondere wird das entsprechende Modul in TUMonline als <span class='fw-bold text-primary'>nicht bestanden (w. Täuschung)</span> markiert, und dur darfst nur noch einen letzten Versuch unternehmen, das Modul zu bestehen (vgl. APSO §24).",
+                "checkForPlagiarism": "Alle deine Einreichungen werden auf Plagiate überprüft. Wenn du beispielsweise Code aus einem Buch oder einer Website kopierst, musst du die Quelle durch eine Anmerkung an den entsprechenden Stellen zitieren, z.B. <span class='fw-bold text-primary'>adaptiert von https://sourcemaking.com/design_patterns/bridge</span>. Andernfalls gilt dies als Plagiat. Das Kopieren von Code von anderen Studierenden ist nicht erlaubt und wird als Plagiat gewertet.",
+                "programmingSubmission": "Beachte, dass die Ergebnisse in den Programmierübungen nur anzeigen, ob deine Einreichung kompiliert oder nicht. Es werden keine tatsächlichen Tests gegen deiner Einreichung ausgeführt. Alle tatsächlichen Tests werden erst nach Abschluss der Klausur durchgeführt. Du kannst nur Punkte für Programmierübungen erhalten, wenn deine Einreichung kompiliert. <span class='fw-bold text-primary'>Kompilierungsfehler führen zu 0 Punkten</span>.",
+                "submissionPeriod": "Um potenzielle technische Probleme zu kompensieren, wird der Abgabezeitraum <span>{{ gracePeriodInMinutes }} Minuten</span> nach dem offiziellen Ende der Klausur geschlossen. Wir empfehlen dir, deine Einreichungen regelmäßig und frühzeitig hochzuladen. <span class='fw-bold text-primary'>Nur deine letzte Einreichung wird bewertet und verspätete Einreichungen werden nicht akzeptiert</span>."
+            },
+            "workingInstruction": {
+                "workingInstructions": "Arbeitsanweisungen:",
+                "important": "<span class='fw-bold text-primary'>Wichtig</span>: Bevor du mit der Lösung einer Aufgabe beginnst, lese die Aufgabenstellung sorgfältig durch.",
```

Comment:
```suggestion
                "important": "<span class='fw-bold text-primary'>Wichtig</span>: Lese die Aufgabenstellung sorgfältig durch, <span class='fw-bold text-primary'>bevor</span> du mit dem Lösen einer Aufgabe beginnst.",
```

Reply:
I think your suggested sentence structure will be a better fit, thanks 👍 But I don't think we need to highlight "bevor", since we highlight the sentence at first with "Wichtig".